lib/drawer/ui/list_module.dart

```diff
@@ -1,20 +1,28 @@
-import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
 import 'package:myecl/drawer/providers/modules_provider.dart';
 import 'package:myecl/drawer/ui/module.dart';
 
-class ListModule extends ConsumerWidget {
+class ListModule extends HookConsumerWidget {
   const ListModule({super.key});
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final modules = ref.watch(listModuleProvider);
-    return SingleChildScrollView(
-      physics: const BouncingScrollPhysics(),
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        children: modules.map((module) => ModuleUI(module: module)).toList(),
+    final scrollController = useScrollController();
+    return Scrollbar(
+      controller: scrollController,
+      interactive: true,
+      radius: const Radius.circular(8),
```

Comment:
Here is how it looks on an Android Device :
![Screenshot_2024-02-03-21-01-50-07_3166164293b8afe0deb3263e48924357.jpg](https://github.com/aeecleclair/Titan/assets/122213719/8cd67fae-2dc7-4321-86ca-7bfcbe367c4f)